spark-connector/src/main/java/com/datastrato/gravitino/spark/plugin/GravitinoDriverPlugin.java

```diff
@@ -0,0 +1,74 @@
+/*
+ *  Copyright 2024 Datastrato Pvt Ltd.
+ *  This software is licensed under the Apache License version 2.
+ */
+
+package com.datastrato.gravitino.spark.plugin;
+
+import com.datastrato.gravitino.spark.GravitinoSparkConfig;
+import com.datastrato.gravitino.spark.catalog.GravitinoCatalog;
+import com.datastrato.gravitino.spark.catalog.GravitinoCatalogManager;
+import com.google.common.base.Preconditions;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Set;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.spark.SparkConf;
+import org.apache.spark.SparkContext;
+import org.apache.spark.api.plugin.DriverPlugin;
+import org.apache.spark.api.plugin.PluginContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * GravitinoDriverPlugin creates GravitinoCatalogManager to fetch catalogs from Gravitino and
+ * register Gravitino catalogs to Spark.
+ */
+public class GravitinoDriverPlugin implements DriverPlugin {
+  private static final Logger LOG = LoggerFactory.getLogger(GravitinoDriverPlugin.class);
+
+  private GravitinoCatalogManager catalogManager;
+
+  @Override
+  public Map<String, String> init(SparkContext sc, PluginContext pluginContext) {
+    SparkConf conf = sc.conf();
+    String gravitinoUri = conf.get(GravitinoSparkConfig.GRAVITINO_URI);
+    String metalake = conf.get(GravitinoSparkConfig.GRAVITINO_METALAKE);
+    Preconditions.checkArgument(
+        StringUtils.isNoneBlank(gravitinoUri),
+        String.format(
+            "%s:%s, should not be empty", GravitinoSparkConfig.GRAVITINO_URI, gravitinoUri));
+    Preconditions.checkArgument(
+        StringUtils.isNoneBlank(metalake),
```

Comment:
What's the difference between `isNotBlank` and `isNoneBlank`?

Reply:
isNoneBlank could  check multi CharSequence, while isNoneBlank check just one CharSequence, it's more suitable to use isNotBlank here, I will update it.

Reply:
done